### ISSUE TITLE
App subcategories should be unorderd lists

### ISSUE THREAD

**Issue by johrstrom** [OSC/ondemand#705]
For accessibility navigation we should probably have every sub-cateogry in the app lists menu panels be actual lists. Probably even every category be lists too.

With the resulting html looking something like this, so it's basically a list of nested lists.

```html
<ul>
  <li>Category
    <ul>Subcategory
      <li>App</li>
      </ul>
    </li>
</ul>
```

Obviously we'd want keep the same look for visual users and keep all the popover descriptions in place, this would just change the semantic structure a bit for keyboard users to step through apps more quickly.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201735133575781/1201737643205667) by [Unito](https://www.unito.io)

**Comment by johrstrom** [comment]
I fixed this just before the 3.1 release. I could track down the commit, but I'm satisfied to just close the ticket.
